src/GraphMetadataViewExtension/ValidationRules/UriValidationRule.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Controls;
+
+namespace Dynamo.GraphMetadata.ValidationRules
+{
+    public class UriValidationRule : ValidationRule
+    {
+        public override ValidationResult Validate(object value, System.Globalization.CultureInfo cultureInfo)
+        {
+            string input = value as string;
+
+            if (String.IsNullOrEmpty(input)) // Valid input, converts to null.
+            {
+                return new ValidationResult(true, null);
+            }
+
+            if (Uri.TryCreate(input, UriKind.Absolute, out Uri outUri))
```

Comment:
I think se should probably allow www.adress.com. Currently we enforce http://www

Reply:
okay, i think the validation becomes kinda pointless then. As it seems `UriKind.RelativeOrAbsolute` will literally accept anything.